lib/index.js

```diff
@@ -59,171 +60,134 @@ bedrock.events.on('bedrock.ready', callback => {
 });
 
 // setup server on bedrock start
-bedrock.events.on('bedrock.start', init);
+bedrock.events.on('bedrock.start', callbackify(async () => init()));
 
-function init(callback) {
-  async.auto({
-    init: callback => bedrock.events.emit(
-      'bedrock-express.init', server, callback),
-    beforeLogger: ['init', callback => {
-      // basic config
-      server.enable('trust proxy');
-      server.disable('x-powered-by');
-      bedrock.events.emit('bedrock-express.configure.logger', server, callback);
-    }],
-    logger: ['beforeLogger', (callback, results) => {
-      if(results.beforeLogger === false) {
-        return callback();
-      }
-      const accessLogger = bedrock.loggers.get('access');
-      server.use(morgan('combined', {
-        stream: {write: str => accessLogger.log('info', str)}
-      }));
-      callback();
-    }],
-    beforeStatic: ['logger', callback => bedrock.events.emit(
-      'bedrock-express.configure.static', server, callback)],
-    static: ['beforeStatic', (callback, results) => {
-      if(results.beforeStatic === false) {
-        return callback();
+async function init() {
+  const emit = promisify(bedrock.events.emit);
```

Comment:
`bedrock.events.emit` already returns a promise ... so this shouldn't be necessary. Did you find that it was?

Reply:
Without `promisify` it's accessing keys of `undefined` and throwing unhandled promise rejections.  I figured there was some calling convention confusion related to callbacks and this fixed it.  I didn't explore further.

Reply:
@davidlehn, we should fix that in bedrock... sorry, another yak to shave. I guess we can accept this for now but add an issue to remove it later once the bug(s) in bedrock are fixed for this method.

Reply:
Fix for the `emit` issue here landed in bedrock@1.17.0.